.claude/commands/gemini-consult.md

```
@@ -0,0 +1,164 @@
+
+**When to use Context7 MCP:**
+- Libraries with frequent updates (e.g., Google GenAI SDK)
+- New libraries you haven't worked with extensively
+- When implementing features that rely heavily on library-specific patterns
+- Whenever uncertainty exists about current best practices
+
+```python
+# Example: Get up-to-date documentation
+library_id = mcp__context7__resolve_library_id(libraryName="google genai python")
+docs = mcp__context7__get_library_docs(
+    context7CompatibleLibraryID=library_id,
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)